Detections/MultipleDataSources/ExchangeServerVulnerabilitiesMarch2021IoCs.yaml

```diff
@@ -139,24 +146,17 @@ query: |
   | extend timestamp = TimeGenerated, IPCustomEntity = cIP, HostCustomEntity = Computer, AccountCustomEntity = csUserName
   ),
   (
-  DeviceNetworkEvents
-  | where (RemoteIPType =~ "Public" and RemoteUrl has_any (domains)) or (isnotempty(RemoteIP) and RemoteIP in (ips))
-  | extend timestamp = TimeGenerated, IPCustomEntity = RemoteIP, HostCustomEntity = DeviceName
-  ),
-  (
-  WindowsFirewall
```

Comment:
We also don't do windows Firewall yet. This is the agent Firewall option, not based on Windows Events.